pkg/kubernetes/scripts/projects.js

```diff
@@ -937,14 +966,20 @@
                     displayName: fields.display.trim(),
                     description: fields.description.trim()
                 };
-
-                return methods.check(request, { "metadata.name": "#project-new-name" })
-                    .then(function() {
-                        return methods.create(request)
-                            .then(function() {
-                                return projectData.shareImages(name, fields.access === "shared");
-                            });
-                    });
+                var chain = $q.when();
+                chain = chain.then(function() {
+                    return methods.check(request, { "metadata.name": "#project-new-name" });
+                });
+                chain = chain.then(function() { 
+                    return methods.create(request);
+                });
+                chain = chain.then(function() { 
+                    return projectData.shareImages(name, fields.access === "shared");
+                });
+                chain = chain.then(function() {
+                    return projectData.makeRegistryAdmin(name, dialogData.currUser);
```

Comment:
Why is this necessary? I don't see that ```oc new-project``` has to perform this action:

```
oc new-project --loglevel=10 blah
oc get policybindings --output=json
```

The above shows scruffy an admin already without being added manually.